botr.py

```diff
@@ -25,51 +25,52 @@ def get_prefix(bot, message):
         for extension in initial_extensions:
             try:
                 bot.load_extension(extension)
+                print(extension +' has been loaded!')
             except Exception as e:
                 print(f'Failed to load extension {extension}.', file=sys.stderr)
                 traceback.print_exc()
+    print('------')
     print(f'Bathbot is fully ready!')
     if not discord.opus.is_loaded():
             discord.opus.load_opus('libopus.so')
             print('Opus has been loaded!')
 
-@bot.command(name='b-load', hidden=True)
-async def cog_load_backup(self, ctx, *, cog: str):
+'''@bot.command(name='load',hidden=True)
```

Comment:
Now that you're in source control, don't comment sections out just straight up delete them. If you need it back you can just check the commit log

Reply:
More old stuff I forgot to remove, agreed.